mxcubeweb/core/components/samplechanger.py

```diff
@@ -440,6 +457,11 @@ def queue_mount_sample(view, data_model, centring_done_cb, async_result):  # noq
                 "sampleID": data_model.loc_str,
             }
 
+            # in this case the sample changer takes the sample from an Harvester
+            # We Harvest the sample and make it ready to load
+            if mount_from_harvester:
```

Comment:
Can't we somehow know from the sample `data_model` that its a harvester sample, like this you dont need `mount_from_harvester`